ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -102,17 +126,38 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    /// Initializes the bottom commanding controller with a given content view controller.
+    /// - Parameter contentViewController: View controller that will be displayed below the bottom commanding UI.
+    @objc public init(with contentViewController: UIViewController?) {
+        self.contentViewController = contentViewController
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
     // MARK: - View building and layout
 
     public override func loadView() {
-        view = BottomSheetPassthroughView()
+        view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
 
         if traitCollection.horizontalSizeClass == .regular {
             setupBottomBarLayout()
         } else {
             setupBottomSheetLayout()
         }
+
+        if let contentViewController = contentViewController,
+           let rootCommandingView = rootCommandingView {
+            addChild(contentViewController)
+            let newContentView: UIView = contentViewController.view
```

Comment:
Probably just let it infer the UIView type?

Reply:
Like the other comment, needed because of the implicitly unwrapped optional.